datafusion/sql/src/planner.rs

```diff
@@ -230,6 +231,41 @@ impl<'a, S: ContextProvider> SqlToRel<'a, S> {
         Ok(Schema::new(fields))
     }
 
+    pub(super) fn build_column_defaults(
```

Comment:
would be nice to explain in comments what exactly gets returned in the tuple for future readers, like `<column name,  expr>`
I'm even thinking should be that be a map instead of `Vec<>`

Reply:
A comment has been added for this.

The main reason is that `Hashmap` has not implemented the `Hash` trait, which is required by `CreateMemoryTable`.